index.rst

```diff
@@ -100,73 +110,268 @@ Data Sets Types and Goals
+    - Essentially only g-band, as there are only a few r-band images available.  This would not then actually satisfy the 2-band MEDIUM color requirement outlined above.
+    - Blind15A_26, Blind15A_40, and Blind15A_42 have been selected for AP testing in
+      https://github.com/lsst/ap_verify_hits2015
+
+MEDIUM
+======
+1. HSC SSP PDR1+PDR2
+    - Planned work to build templates from PDR1 and then run subtractions from the new data in PDR2 from later years.
+    https://jira.lsstcorp.org/browse/DM-20559
+    https://jira.lsstcorp.org/browse/DM-20560
+
+It's less clear that it's important to do active regular testing of DIA on LARGE datasets.  MEDIUM should be sufficient to characterize the key science performance goals.
+
+
+AP Candidate Additional Datasets
+================================
+1. DECam DES SN fields
+    - 8 shallow SN fields, 2 deep SN fields
+    - griz observation sequences obtained ~ weekly
+    - deep fields have multiple exposures in one field in the same filter each night, with other filters other nights; shallow fields have a single griz sequence in one night.  Former is more LSST-like.
+    - Raw data are public
+    - 10 fields from 2014 (DES Y2) in field SN-X3.
+    - g (no particular reason for this choice)
+    - visits = [371412, 371413, 376667, 376668, 379288, 379289, 379290, 381528, 381529]
+    - Available on lsst-dev in `/datasets/des_sn`
+
+2. HSC New Horizons
+    - crowded stellar field (Galactic Bulge)
+    - available to us (not fully public?); unclear details of numbers of epochs, etc.
+    - scientifically untapped
+    - Available on lsst-dev at `/datasets/hsc/raw/newhorizons/`
+
+3. DECam Bulge survey
+    - crowded stellar field
+    - Propoasal ID 2013A-0719 (PI Saha)
+    - limited publications to date: 2017AJ....154...85V; total boundaries of survey unclear.
+    - published example shows that globular cluster M5 field has 50+ observations over 2+ seasons in each of ugriz
+
+4. DECam NEO survey
+    - PI L. Allen
+    - 320 square degrees; 5 epochs a night in a single filter with 5 minute cadence, repeating for three nights
+    - 3 seasons of data
+
+5. HSC SSP Deep or Ultra-Deep:
+    - grizy; exposure times 3-5 minutes; tens of epochs available
+    - two UD fields and 15 deep fields
+    - Open Time observations from Yoshida
+    - tens of epochs over a couple of nights for a range of fields
+    - GAMA09 and VVDS overlap SSP wide (only) but Yoshida reports the seeing was bad (~1")
+
+
+====================================
+Datasets considered but not selected
+====================================
+ * CFHT-SNLS
+   - Suitable for some AP performance.  But reason to select CFHT over DECam.
+ * CFHTLS-Deep
+   - Suitable, but no obvious reason to select CFHT over DECam
+ * PTF
+   - Tens to thousands of epochs of public images available in two filters (g & R), but camera characteristics are markedly different–2"+ seeing, 1" pixels, and much shallower.
+ * ZTF
+   - Same sampling issues as PTF.
```

Comment:
obs_ztf now exists, but has not been thoroughly tested; not all desired calibration products seem to be publicly available.